pfb-analysis/import/import_neighborhood.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 NB_INPUT_SRID="${NB_INPUT_SRID:-4326}"
-NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-4326}"
-NB_BOUNDARY_BUFFER="${NB_BOUNDARY_BUFFER:-0}"
+NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-2163}"
```

Comment:
Whoops, I misspoke. 2163 seems to make the most sense here since we can then leave the default max trip distance in meters and the resulting analysis from the defaults would make sense.

 I think I probably chose 3600 since its an even number very close to 11000 when multiplied by 3, which is an approximation but not exact for ft -> m. So I'm fine with doing 3300. 

So summary of changes:
- Update all defaults/comments in the analysis to use NB_OUTPUT_SRID=2163
- Do the same for NB_MAX_TRIP_DISTANCE=3300